packages/plugin-angular/src/index.ts

```diff
@@ -42,6 +54,20 @@ export class BugsnagErrorHandler extends ErrorHandler {
 
 const plugin: Plugin = {
   load: (client: Client): ErrorHandler => {
+    const originalNotify = client._notify
+
+    client._notify = function (event) {
+      if (isNgZoneEnabled) {
+        // run notify in the root zone to avoid triggering change detection
+        Zone.root.run(() => {
+          originalNotify(event)
+        })
+      } else {
+        // if zones are not enabled, change detection will not run anyway
+        originalNotify(event)
```

Comment:
[`_notify` actually has more than one argument](https://github.com/bugsnag/bugsnag-js/blob/next/packages/core/client.js#L292); probably best to use `arguments` to avoid having to update multiple places if they change

```suggestion
    client._notify = function () {
      const notifyArguments = arguments
      if (isNgZoneEnabled) {
        // run notify in the root zone to avoid triggering change detection
        Zone.root.run(() => {
          originalNotify(notifyArguments)
        })
      } else {
        // if zones are not enabled, change detection will not run anyway
        originalNotify(notifyArguments)
```